packages/tasks/src/tasks/zero-shot-object-detection/data.ts

```diff
@@ -0,0 +1,57 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "object-detection-input.jpg",
```

Comment:
Would you like to add the image in this PR?

Reply:
I was not sure about this part. Could you guide me how to do it?

Reply:
Sure! You can just open a PR to this repository with the folder "zero-shot-object-detection" and then the filename you contributed you can write it to `filename` https://huggingface.co/datasets/huggingfacejs/tasks

Reply:
Thanks a lot! Will work on it asap

Reply:
I've just created the PR on the dataset